K-diff Pairs in an Array.py

```diff
@@ -0,0 +1,34 @@
+"""532. K-diff Pairs in an Array
+Time Complexity - O(n)
+Space Complexity - O(1)
+Sort array
+Consider 2 pointers 1 fast,1 slow
+while slow < len(nums) and fast < len(nums)
+    if slow == fast or nums[fast] - nums[slow] < k
+        fast ++
+    elif nums[fast] - nums[slow] > k
+        slow++
+    else
+        count ++
+        slow++
+        fast ++
+        while loop to prevent duplicates
+return count"""
+class Solution:
```

Comment:
Good Job! Diksha. Keep it up. Please add some comments in plain English to explain your approach.